superset/migrations/shared/migrate_viz/processors.py

```diff
@@ -189,3 +163,24 @@ def _pre_action(self) -> None:
         elif line_interpolation == "step-after":
             self.target_viz_type = "echarts_timeseries_step"
             self.data["seriesType"] = "end"
+
+
+class MigrateAreaChart(TimeseriesChart):
+    source_viz_type = "area"
+    target_viz_type = "echarts_area"
+    stacked_map = {
+        "expand": "Expand",
+        "stack": "Stack",
+        "stream": "Stream",
+    }
+
+    def _pre_action(self) -> None:
+        super()._pre_action()
+
+        self.remove_keys.add("stacked_style")
+
+        self.data["stack"] = self.stacked_map.get(
```

Comment:
Should the `stack` key be `None` or undefined if the resulting key isn't in `self.stacked_map`?

Reply:
Yes. It can be `None` or `undefined`.

Reply:
Do we have a preference or whether we explicitly define `None`? I guess there's likely so many inconsistencies on both the front- and back-end with regards to when/where to persist values that even if we make it undefined (and thus minimizing the payload) there's no guarantee that the field wont materialize with a value of `None` on save.

Reply:
Exactly 😆